scss/partials/_faq.scss

```diff
@@ -0,0 +1,156 @@
+
+.faq{
+
+  .transition{
+  transition: all 0.25s ease-in-out;

```

Comment:
keep your selectors more consistent using the same spaces after a `property`, the same indentation as well etc..